doc/user/content/manage/user-management/rbac-intro.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Role-based access management"
+description: "How Materialize implements RBAC"
+menu:
+  main:
+    parent: user-management
+    weight: 10
+---
+
+This page introduces role-based access management (RBAC) in Materialize. RBAC
+allows you to apply granular privileges to your Materialize objects and clusters. Organizations
+using RBAC can manage user roles and privileges to ensure there is not
+unauthorized or improper access to sensitive objects.
+
+In Materialize, RBAC allows organization administrators to:
+
+* Determine which users have read or write permissions for specific objects
+
+* Control how users interact with clusters by giving them different levels of access to
+resources
+
+* Prevent accident operations from unauthorized users
+
+* Isolate access to user-facing data from internal organization data
+
+Materialize object access is also dependant on cluster privileges.
+Users who need access to an object like a schema or a database must also have
+the same level of access to the cluster.
+
+!(#TODO_Diagram)
```

Comment:
What will this diagram show? If it won't be added until a follow-up PR, let's comment this out.